posthog/models/team/team.py

```diff
@@ -120,6 +120,29 @@ def create_with_data(self, *, initiating_user: Optional["User"], **kwargs) -> "T
                 filters=playlist["filters"],
                 description=str(playlist.get("description", "")),
             )
+
+        # Create GeoIP transformation if MMDB is enabled
+        if not settings.DISABLE_MMDB:
+            from posthog.models.hog_functions.hog_function import HogFunction
+            from posthog.plugins.plugin_server_api import get_hog_function_template
+
+            response = get_hog_function_template("template-geoip")
```

Comment:
Rather use the `HogFunctionsTemplates` class that already takes care of caching and loading